source/docs/software/wpilib-tools/pathweaver/integrating-robot-program.rst

```diff
@@ -5,6 +5,8 @@ The ``TrajectoryUtil`` class can be used to import a PathWeaver JSON into robot
 
 The ``fromPathweaverJson`` (Java) / ``FromPathweaverJson`` (C++) static methods in ``TrajectoryUtil`` can be used to create a trajectory from a JSON file stored on the roboRIO file system.
 
+.. important:: To be compatible with the ``Field2d`` view in the simulator GUI, the coordinates for the exported JSON have changed. Previously (before 2021), the range of the y-coordinate was from -27 feet to 0 feet whereas now, the range of the y-coordinate is from 0 feet to 27 feet. This should not affect teams who are correctly :ref:`resetting their odometry to the starting pose of the trajectory <docs/software/examples-tutorials/trajectory-tutorial/creating-following-trajectory:Creating the RamseteCommand>` before path following.
```

Comment:
I think it would be helpful to say which coordinate is the top of the screen and which is bottom.